ui-v2/app/styles/components/composite-row.scss

```diff
@@ -13,6 +13,9 @@
 .consul-role-list > ul > li:not(:first-child) {
   @extend %with-composite-row-intent;
 }
+.consul-lock-session-list ul > li:not(:first-child) {
+  @extend %with-one-action-row;
+}
```

Comment:
I wonder if we could change this to use the new slide-y confirmation actions we use everywhere else? It would just be a single item menu I think? Maybe check with the others and see what they think? (Let me know if I'm not being clear what I mean here).

Actually, I think we have an item on our backlog to replace the remaining confirmation dialogs like this with the new slide-y ones.